archive/r/rust/fibonacci.rs

```diff
@@ -0,0 +1,37 @@
+const LIMIT: i64 = 92;
```

Comment:
If you used an unsigned 64 bit integer, you could increase the limit to 93.

Reply:
Done